src/rust/bitbox02-rust/src/hww/api/bluetooth.rs

```diff
@@ -173,10 +173,33 @@ async fn process_upgrade(
     } else {
         hal.ui().status("Upgrade failed", false).await;
     }
-
     response
 }
 
+async fn process_toggle_enabled(hal: &mut impl crate::hal::Hal) -> Result<Response, Error> {
+    let enabled = memory::ble_enabled();
+    let body = if enabled {
+        "Turn off blueooth?"
```

Comment:
Typo in bluetooth. Also it should be uppercase Bluetooth right?

Not sure, but maybe better to put it on two lines so the confirmation arrows don't go over the text.

Reply:
How about just "Enable?" / "Disable?"? Bluetooth is already in the title.